entries/entry03.md

```diff
@@ -1,7 +1,15 @@
 # Entry 3
-##### X/X/XX
+##### 1/5/20
+
+## Engineering Design Process:
+For the past few weeks, I have shifted most of my focus from [Android Studio](https://developer.android.com/docs) to creating the content of my app. Knowing that there will be a considerable amount of content that I will have to create since I want to be as specific as possible with everything about [PUBG](https://documentation.pubg.com/en/index.html), and that content will take up the most amount of work as opposed to actually making the app on Android Studio, I have decided to halt the learning process of Android Studio for later and focused more on creating the content of my app on a [Google Doc](https://www.google.com/docs/about/). I also did this because I prefer to be consistent with the project by focusing on one thing at a time, rather than being inconsistent and bouncing back and forth with creating the content and tinkering with Android Studio. I've also been playing PUBG more often than usual so that I can see what tips and other things I can find to include in my app, since most of the content will be from my own personal experience of playing the game. Lastly, I have recorded and edited all of the gun audio from PUBG for my PUBG gun sound quiz with [Audacity](https://www.audacityteam.org/help/documentation/). 
+
+![doc](../doc.png)
+![audio](../audio.png)
+
+## Knowledge and Skills:
```

Comment:
What are you learning about **Android** Java? (that applies to your project). Let's see some code snippets! It's fine to work on content, but it's going to be pointless if you don't make progress on Android Studio.

Reply:
Also, you didn't mention any specific skills here (see the skill list).